app/src/main/java/ca/bc/gov/bchealth/ui/healthpass/HealthPassFragment.kt

```diff
@@ -172,6 +184,11 @@ class HealthPassFragment : Fragment(R.layout.fragment_helath_pass) {
             if (uiState.isAuthenticationRequired) {
                 findNavController().navigate(R.id.biometricsAuthenticationFragment)
             }
+
+            if (uiState.isBcscLoginRequiredPostBiometrics) {
+                findNavController().navigate(R.id.action_healthPassFragment_to_bcscAuthInfoFragment)
```

Comment:
You need to call BCSC auth directly no action definition is required

Reply:
this is taken care